src/controllers/interaction.ts

```diff
@@ -51,6 +54,10 @@ export const interactionStartControllerFactory =
         setLoginHintInUnauthenticatedSession(req, login_hint);
       }
 
+      if (siret_hint) {
```

Comment:
issue: I doubt that `siret_hint` is validated by the `oidc-provider` library.
We should explicitly validate that `siret_hint` is a non-empty string of 14 digits before storing it in the session.